sagemaker-debugger/debugger_interactive_analysis_profiling/interactive_analysis_profiling_data.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Extra space after <code>profiler</code>  </p><p> </p><p>Conclusion link may not be working?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Yes, it looks ok on my NI. For some reason the ReviewNB is not properly working for these links.</p>